### DISCUSSION TITLE
Are completion item kind highlights supported?

### DISCUSSION THREAD

**Discussion by arafatamim** [neoclide/coc.nvim#3554] [Ideas]
Looks like this (from `nvim-cmp`:
![image](https://user-images.githubusercontent.com/31634638/148589531-bec0b644-ad0c-4ec1-a2f0-2d10c0304281.png)
`nvim-cmp` has highlight groups like `CmpItemKindConstant`, `CmpItemKindFunction` to allow coloring the individual completion kinds.

**Comment by oblitum** [comment]
It's not, `nvim-cmp` builds custom completion menu, unrelated to the vim/nvim native one. See https://github.com/neoclide/coc.nvim/issues/3480. You may feature request, as stated in that thread.

**Comment by chemzqm** [comment]
Not yet.
